### ISSUE TITLE
Institution Status is missing many fields

### ISSUE THREAD

**Issue by danieldides** [plaid/plaid-java#205]
Using the Java client, hitting the `POST /institutions/get_by_id` with the optional `include_status` field set returns an incomplete response. With the Java client, only the "item_logins" field of the status is returned whereas the actual API response returns many more. I believe this is because there are fields missing on the POJO the API response is de-serialized into.

# Examples (hitting the same institution)
## Example API response hitting the API directly:

```json
"status": {
      "auth": {
        "breakdown": {
          "error_institution": 0.05,
          "error_plaid": 0.007,
          "success": 0.942
        },
        "last_status_change": "2020-04-01T13:56:59Z",
        "status": "HEALTHY"
      },
      "balance": {
        "breakdown": {
          "error_institution": 0.75,
          "error_plaid": 0,
          "success": 0.25
        },
        "last_status_change": "2020-04-01T21:48:58Z",
        "status": "DEGRADED"
      },
      "identity": {
        "breakdown": {
          "error_institution": 0,
          "error_plaid": 0,
          "success": 1
        },
        "last_status_change": "2020-04-01T22:00:58Z",
        "status": "HEALTHY"
      },
      "item_logins": {
        "breakdown": {
          "error_institution": 0.297,
          "error_plaid": 0.014,
          "success": 0.689
        },
        "last_status_change": "2020-04-01T21:28:58Z",
        "status": "HEALTHY"
      },
      "transactions_updates": {
        "breakdown": {
          "error_institution": 0,
          "error_plaid": 0,
          "refresh_interval": "STOPPED",
          "success": 1
        },
        "last_status_change": "2020-04-01T22:07:58Z",
        "status": "DOWN"
      }
    }
  },
```

## Example response with the Java client
```json
 "status": {
            "itemLogins": {
                "status": "HEALTHY",
                "lastStatusChange": 1585776538000,
                "breakdown": {
                    "success": 0.689,
                    "errorPlaid": 0.014,
                    "errorInstitution": 0.297
                }
            }
        },
```

**Comment by davidzhanghp** [comment]
addressed by https://github.com/plaid/plaid-java/pull/206

also @danieldides we'll likely release a new version of java next week so this will make it into that version

**Comment by skylarmb** [comment]
This has been released as part of 5.7.0
